app/controllers/extra_locales_controller.rb

```diff
@@ -12,30 +12,36 @@ class ExtraLocalesController < ApplicationController
   OVERRIDES_BUNDLE = "overrides"
   MD5_HASH_LENGTH = 32
   MF_BUNDLE = "mf"
-  BUNDLES = [OVERRIDES_BUNDLE, MF_BUNDLE]
+  ADMIN_BUNDLE = "admin"
+  WIZARD_BUNDLE = "wizard"
+
+  SITE_SPECIFIC_BUNDLES = [OVERRIDES_BUNDLE, MF_BUNDLE]
+  SHARED_BUNDLES = [ADMIN_BUNDLE, WIZARD_BUNDLE]
 
   class << self
     def js_digests
-      @js_digests ||= {}
+      @js_digests ||= { site_specific: {}, shared: {} }
```

Comment:
Split this up because technically there was a possibility of a clash in the past, if a site was named "admin" or "wizard"